all/docs/js.md

```diff
@@ -605,23 +599,103 @@ Userfront.init("demo1234");
 Userfront.resetPassword({
   password: "myshinynewpassword",
 });
+```
+
+::::
+:::::
+
+## updatePassword (options)
+
+::::: row
+:::: left
+
+Updates a user's password with one of the following methods:
+
+- Reset link credentials (`token` and `uuid`); or
```

Comment:
I suggest "The reset link credentials". As written it's easy to read _reset_ as a verb, I did a double take.